privacy-fileinjection-maven-plugin/src/test/java/io/konig/privacy/injection/PrivacyInjectionTest.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * 
+ */
+package io.konig.privacy.injection;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+
+import org.junit.Test;
+
+
+public class PrivacyInjectionTest {
+
+	@Test
+	public void testInjection() {
+		File file = new File("src/test/resources/cloudformation.yaml");
+		assertTrue(file.exists());		
+		assertContainsParameters(file,"development");
+		assertContainsParameters(file,"Alan Turing");
+		
+	}
+	
+	public void assertContainsParameters(File file, String inputString){
```

Comment:
Change this method so that it receives a parsed YAML object instead of a File.  Verify that the expected property in the YAML file has the expected value instead of just scanning the entire file for the presence of the input string.